src/components/PostView.vue

```diff
@@ -28,7 +28,8 @@
     mixins: [postCat],
     data () {
       return {
-        'catUrl': null
+        'catUrl': null,
+        'title' : null
```

Comment:
you can remove the blank space before ':' according to the coding style :smile_cat: